indexer/workers/importer.py

```diff
@@ -124,8 +126,9 @@ def index_routing(self, publication_date_str: Optional[str]) -> str:
                     year = -1
             except ValueError as e:
                 logger.warning("Error parsing date: '%s" % str(e))
-
-        index_name_prefix = self.index_name_prefix
+        index_name_prefix = self.index_name_prefix or os.environ.get(
+            "ELASTICSEARCH_INDEX_NAME_PREFIX"
+        )
```

Comment:
```suggestion
        index_name_prefix = self.index_name_prefix
```

[`os.environ`](https://github.com/mediacloud/story-indexer/blob/c19aea80e41abaca01db094d2014cce8046ffb7d/indexer/workers/importer.py#L90-L96) has already been used to set the default value here.